### PR TITLE
Add chart

### PR DESCRIPTION
**Pull request by pawalt**

### CODE REVIEW

File: chart/templates/tests/test-connection.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+# lol no
```
Comment (ArmaanT) [inline chart/templates/tests/test-connection.yaml]: I know we're not going to add tests, so we may as well delete this file.

File: main.go [hunk 2]
```diff
@@ -33,56 +32,53 @@ func main() {
...
+			} else {
+				log.Infof("Successfully bounced ingress %s in namespace %s.\n", ingress.Name, ingress.Namespace)
 			}
 		}
-
-		log.Infof("Sleeping at time %s for 5 minutes\n", time.Now().String())
-		time.Sleep(5 * time.Minute)
 	}
+
+	log.Infof("Bouncing completed 💯")
```
Comment (ArmaanT) [inline main.go:81]: If Unicode causes our logs to break, I'm going to be mad

File: .circleci/config.yml [hunk 3]
```diff
@@ -1,10 +1,20 @@
 version: 2.1
 orbs:
   docker: circleci/docker@0.5.19
+  helm-tools: pennlabs/helm-tools@0.1.1
 workflows:
   version: 2
   build-and-publish:
     jobs:
+      - helm-tools/helm-publish:
```
Comment (ArmaanT) [inline .circleci/config.yml]: I think we should move this job after `docker/publish` and require that job to finish first.

File: chart/values.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+ingress-svc:
+  name: traefik-ingress-service
+  namespace: kube-system
+
+image:
+  repository: pennlabs/ingress-bouncer
+  pullPolicy: IfNotPresent
+  tag: latest
```
Comment (ArmaanT) [inline chart/values.yaml:8]: Can we somehow specify to use the sha of the current commit?

**Review by ArmaanT — CHANGES_REQUESTED** [review]

File: chart/templates/tests/test-connection.yaml (hunk 1 above)
Comment (pawalt) [reply chart/templates/tests/test-connection.yaml]: fair

File: main.go (hunk 2 above)
Comment (pawalt) [reply main.go:81]: it wont

File: .circleci/config.yml (hunk 3 above)
Comment (pawalt) [reply .circleci/config.yml]: oh nice

File: chart/values.yaml (hunk 4 above)
Comment (pawalt) [reply chart/values.yaml:8]: nah that shouldn't be the job of the chart anyway. the chart should specify latest, and the user should specify if they want to override

Comment (ArmaanT) [reply chart/values.yaml:8]: Okay, makes sense

**Review by ArmaanT — APPROVED** [review]
LGTM!

**Pull request merged**